src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -91,18 +97,18 @@ def load_mlflow_model(
             raise UserConfigValidationException(
                 "Unable to retrieve model by model id {} in workspace {}, error:\n{}".format(
                     model_id, workspace.name, e
-                )
+                ), e
             )
         model_uri = "models:/{}/{}".format(model.name, model.version)
 
     if use_model_dependency:
         try:
             pip_file = mlflow.pyfunc.get_model_dependencies(model_uri)
         except Exception as e:
-            raise ValueError(
+            raise UserConfigError(
```

Comment:
we are converting load_mlflow_model error to UserConfigError? If this is automl model non-support error, can we tell the user directly 'at this moment it is not supported'?

Reply:
this error is raised on get_model_dependencies() api call. This api is expected to work for proper mlflow input. This is not related to any automl specific error.